solidity/random-beacon/README.adoc

```diff
@@ -197,11 +196,10 @@ rewards.
 [[fees-and-rewards]]
 === Fees and Rewards
 
-Relay requester should provide a fee in T. The value of the fee is a governable
-parameter. The entire fee is deposited in the DKG rewards pool that is used to
-reimburse for different actions related to DKG.
+Relay requester should provide a fee in T. The entire fee is deposited in the DKG
+rewards pool that is used to reimburse for different actions related to DKG.
```

Comment:
The fee for submitting a relay request will be governable, right? Isn't it missing it from the list?